tools/acorn-optimizer.js

```diff
@@ -1830,7 +1838,8 @@ try {
     // Keep in sync with --language_in that we pass to closure in building.py
     ecmaVersion: 2020,
     preserveParens: closureFriendly,
-    onComment: closureFriendly ? sourceComments : undefined
+    onComment: closureFriendly ? sourceComments : undefined,
+    sourceType: exportES6 ? "module" : "script"
```

Comment:
Maybe add a trailing `,` here so that adding new lines in the future doesn't create more diff than needed?